src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -1066,6 +1066,64 @@ SELECT relname FROM pg_class WHERE relname LIKE 'test_table_1%';
 (0 rows)
 
 \c - - - :master_port
+-- verify logged info is propagated to workers when distributing the table
```

Comment:
Could we verify these commands with `citus.log_remote_commands`? Have you considered that?

Reply:
I had not considered it but makes sense 👍

Reply:
Update: keeping the first approach for consistency with the old `multi_alter_table_statements` source